Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -227,6 +245,8 @@ extension ActivityStreamPanel {
             return configureTopSitesCell(cell, forIndexPath: indexPath)
         case .Highlights:
             return configureHistoryItemCell(cell, forIndexPath: indexPath)
+        case .HighlightIntro:
+            return configureHighlightIntroCell(cell, forIndexPath: indexPath)
```

Comment:
I wanted to keep a consistent style. And this way if we want to add anything later its already structured correctly.